test/ping/conversation_test.rb

```diff
@@ -36,4 +36,33 @@ def test_send_message
 
     assert_equal "d0c7a2e6-8084-4e79-8483-e4a1352b81f7", message.id
   end
+
+  def test_successful_delivery
+    fake("api/ping-api/v1/conversations/123/messages/111/delivery_confirmation",
+      method: :post,
+      body: load_fixture('ping/successful_delivery_confirmation'))
+
+    conversation = ShopifyAPI::Ping::Conversation.new(id: '123')
+    delivery_confirmation = conversation.successful_delivery(
+      message_id: '111',
+      delivery_timestamp: Time.now.to_s
+    )
+
+    assert_equal("true", delivery_confirmation.delivered)
```

Comment:
Should we assert that `message_id` and `delivery_timestamp` are set?

Reply:
`message_id` is actually used as a variable in the endpoint for the delivery confirmation resource